src/sentry/models/grouptombstone.py

```diff
@@ -39,6 +40,8 @@ class GroupTombstone(Model):
         blank=True, null=True
     )
     actor_id = BoundedPositiveIntegerField(null=True)
+    times_seen = BoundedPositiveIntegerField(db_default=0, null=True)
+    last_seen = models.DateTimeField(db_default=Now(), null=True)
```

Comment:
If you set `db_default` here, all existing rows will have a default of now. You'll need to set this to just be `default` like it was originally.